src/XrdHttpTpc/XrdHttpTpcTPC.cc

```diff
@@ -966,15 +999,19 @@ int TPCHandler::ProcessPullReq(const std::string &resource, XrdHttpExtReq &req)
     {
         //Get the content-length of the source file and pass it to the OSS layer
         //during the open
-        bool success;
-        GetContentLengthTPCPull(curl, req, sourceFileContentLength, success, rec);
+        bool success = false;
+        bool mismatchDigests = false;
+        std::map<std::string,std::string> sourceFileReprDigest;
+        GetRemoteFileInfoTPCPull(curl, req, sourceFileContentLength, sourceFileReprDigest, success, rec);
```

Comment:
This function should probably return `bool` instead of taking `success` as input, what do you think? Then you'd pass a variable in to get the size (which should always be used, I see here it's not being used from the return value).